static/js/ducks/profile.js

```diff
@@ -52,7 +52,9 @@ const initialState = {
   roles: [],
   acls: [],
   tokens: [],
-  preferences: {}
+  preferences: {
+    newsFeed: {}
```

Comment:
This seems unnecessary, and I'm not sure we want to have to keep updating this default value every time we add a new configurable component/widget. The `NewsFeed` component already checks whether `preferences` has  a `newsFeed` attr, and supplies a default value if not. I think it'd simplify the process of adding new features to just have `preferences` default to `{}` here.

Reply:
Removed, let's see how the tests go!